image_explorer/public/js/image_explorer.js

```diff
@@ -114,7 +121,7 @@ function ImageExplorerBlock(runtime, element) {
       var close_btn = ".image-explorer-close-reveal";
       var clicked_outside_feedback = (target.closest('.image-explorer-hotspot-reveal').length === 0);
       if (target.is(close_btn) || clicked_outside_feedback) {
-        close_feedback();
+          close_feedback();
```

Comment:
Does the indentation level need to be increased here?
It no longer matches the rest of this block.